test/auro-menu.test.js

```diff
@@ -1,4 +1,4 @@
-import { fixture, html, expect, oneEvent, elementUpdated } from '@open-wc/testing';
+import { fixture, html, expect, oneEvent, elementUpdated, aTimeout } from '@open-wc/testing';
```

Comment:
`aTimeout` was added here but it's never used in the test file.

```suggestion
import { fixture, html, expect, oneEvent, elementUpdated } from '@open-wc/testing';
```

Reply:
sure, in a sec.